### PR TITLE
2.10.1

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: javascript.ru/2.10.1/2.10.1.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿alert(123 ^ 0); // 123
+alert(0 ^ 123); // 123
+alert(~~123); // 123
```
Comment (KasichMax) [inline javascript.ru/2.10.1/2.10.1.js]: Do output like `alert('123 ^ 0 = ' + 123 ^ 0)`

Comment (akravchenko564) [reply javascript.ru/2.10.1/2.10.1.js]: Done 1598cc2

File: .vs/config/applicationhost.config [hunk 2]
```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (KasichMax) [inline .vs/config/applicationhost.config]: Remove this file and add it to ignore list.

Comment (KasichMax) [inline .vs/config/applicationhost.config]: Remove this file and add it to ignore list.

Comment (akravchenko564) [reply .vs/config/applicationhost.config]: Redundant file has been removed 7ebce11 .

**Comment by KasichMax** [comment]
OK. Looks good, but next time, please, wait for an approve before merging. Sometimes after you fix a comment there are can be one more. Also now I can't post approve.

**Pull request merged**